types/jsreport-core/index.d.ts

```diff
@@ -15,20 +18,36 @@ declare namespace JsReport {
     type Recipe = "html";
 
     interface Template {
-        content: string;
+        content?: string;
```

Comment:
`?` is not needed here.

Reply:
my preference would be to make this optional as I don't see where it's "required" and I have several reports that utilize the same engine, helpers, recipe and only differ on the content. Rather than having 5 different template objects I would prefer to do something like:

`
const main_template: JsReport.Template = { engine: 'handlebars', helpers: '....', recipe: 'xlsx' };
const daily_report_template: JsReport.Template = { content: '...daily...', ...main_template };
const monthly_report_template: JsReport.Template = { content: '...monthly...', ...main_template };
`